backend/src/test/java/dev/tripdraw/presentation/controller/PostControllerTest.java

```diff
@@ -338,6 +340,56 @@ void setUp() {
                 .statusCode(BAD_REQUEST.value());
     }
 
+    @Test
+    void 특정_감상을_조회한다() {
+        // given
+        PostCreateResponse postResponse = createPost();
+
+        // when
+        ExtractableResponse<Response> findResponse = RestAssured.given().log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .auth().preemptive().oauth2(통후추_BASE64)
+                .when().get("/posts/" + postResponse.postId())
+                .then().log().all()
+                .extract();
+
+        PostResponse getResponse = findResponse.as(PostResponse.class);
+
+        // then
+        assertSoftly(softly -> {
+            softly.assertThat(findResponse.statusCode()).isEqualTo(OK.value());
+            softly.assertThat(getResponse.postId()).isNotNull();
+            softly.assertThat(getResponse.title()).isEqualTo("우도의 바닷가");
+            softly.assertThat(getResponse.pointResponse().pointId()).isNotNull();
+            softly.assertThat(getResponse.pointResponse().latitude()).isEqualTo(1.1);
+        });
+    }
+
+    @Test
+    void 특정_감상을_조회할_때_인증에_실패하면_예외가_발생한다() {
+        // given
+        PostCreateResponse postCreateResponse = createPost();
+
+        // expect
+        RestAssured.given().log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .auth().preemptive().oauth2(순후추_BASE64)
+                .when().get("/posts/" + postCreateResponse.postId())
+                .then().log().all()
+                .statusCode(FORBIDDEN.value());
+    }
+
+    @Test
+    void 특정_감상을_조회할_때_존재하지_않는_감상_ID이면_예외가_발생한다() {
+        // given & expect
+        RestAssured.given().log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .auth().preemptive().oauth2(통후추_BASE64)
+                .when().get("/posts/-1")
+                .then().log().all()
+                .statusCode(NOT_FOUND.value());
+    }
```

Comment:
URL을 `/posts/{postId}`로 명시하고, 파라메터로 path variable을 설정하면 테스트 코드 읽기가 더욱 편할 것 같습니다!
```suggestion
    @Test
    void 특정_감상을_조회할_때_존재하지_않는_감상_ID이면_예외가_발생한다() {
        // given & expect
        RestAssured.given().log().all()
                .contentType(MediaType.APPLICATION_JSON_VALUE)
                .auth().preemptive().oauth2(통후추_BASE64)
                .when().get("/posts/{postId}", -1L)
                .then().log().all()
                .statusCode(NOT_FOUND.value());
    }
```

Reply:
오 너무 좋은데요? 바로 다 반영해버렸습니다